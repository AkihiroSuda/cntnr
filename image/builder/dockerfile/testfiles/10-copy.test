FROM alpine:3.7
COPY entrypoint.sh /entrypoint.sh
COPY --chown=root:root entrypoint.sh *.conf /
COPY --from=0 file1 file2 /
COPY --from=0 --chown=root:root file1 file2 /