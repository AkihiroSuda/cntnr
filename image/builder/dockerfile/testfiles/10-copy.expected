FROM alpine:3.7
NAME 0

COPY dir="./ctx" "entrypoint.sh" "/entrypoint.sh" nil
COPY dir="./ctx" "entrypoint.sh" "*.conf" "/" root:root
COPY image="0" "file1" "file2" "/" nil
COPY image="0" "file1" "file2" "/" root:root